src/index.js

```diff
@@ -748,10 +779,51 @@ class CgSlider extends EventEmitter {
     this._updateAriaLimits();
     this._updateAriaLabels();
     this._updateDisabled();
+    this._updateTicks();
 
     this.container.appendChild(this._rootElement);
   }
 
+  /**
+   * Get percentage offset & add ticks
+   * @private
+   */
+  _updateTicks() {
+    if (!this._ticksElement)
+      return;
+    
+    helpFuncs.removeChildElements(this._ticksElement);
+    
+    if (this._settings['ticks']) {
```

Comment:
**Some code style remarks**
1) Property `this._settings['ticks']` used in this method several times. Please declare separate variable for it.
2) Please use dot notation if there is no necessity of using brackets:
`this._settings.ticks`
3) Also code would looked and read better if it were less nested `if` (and other) statements. To avoid it in this place you can add `return` after `if` if requirement is not met (like in very beggining of your method):
```
if (!this._settings.ticks)
  return;

// rest code here without nesting into if statement
```